templates/index.html

```diff
@@ -7,8 +7,25 @@
 
 {% block takeover_content %}
 
+{% include "takeovers/_rigado_webinar.html" %}
 {% include "takeovers/_private_cloud_economics.html" %}
 
+<script>
+  if (window.localStorage && window.sessionStorage) {
+    if (localStorage.getItem('TAKEOVER_SEEN')) {
+      if (!sessionStorage.getItem('TAKEOVER_SEEN')) {
```

Comment:
It will be on the next session as the `sessionStorage` will have been cleared. This ensures that the newer takeover doesn't disappear if the user visits another page and comes back to the homepage in the same session.